### DISCUSSION TITLE
How do I change the PG REST URL for local development when starting supabase?

### DISCUSSION THREAD

**Discussion by JJosephttg** [supabase/supabase#9153] [Questions] [answered]
I am currently trying to see how my PWA application is working on mobile devices and I cannot connect to the API in local development because it is hosted only on the machine that is serving the API. Is there a way to expand supabase to be shown on the network so that my front end app can connect to it from a client on a different machine on the same network? I got around the app on the front end using localhost by using window.location, however the actual API itself seems to be hosted on localhost rather than the IP.

The documentation is very limited on this, and I was looking at the config.toml but there isn't a variable/property for the host

**Comment by Sukikiroi** [comment, marked as answer]
Hello i did the same with just static IP from the server machin
![image](https://user-images.githubusercontent.com/47844266/192203127-f26a759a-8e86-4e67-af4f-fb472c8c287f.png)

**Reply by Ziinc** [reply]
@JJosephttg  When exposed on localhost, you will be able to connect to that machine when on the same network (assuming no local firewall).
As this is a networking issue for a non-standard dev workflow, it would not be appropriate for it to be in the supabase docs.
